ProteoformSuiteInternal/NeuCodePair.cs

```diff
@@ -30,17 +30,17 @@ public NeuCodePair(Component neuCodeLight, Component neuCodeHeavy, double mass_d
             double firstCorrection;
 
             if (light_is_lower)
-                firstCorrection = neuCodeLight.corrected_mass + diff_integer * Lollipop.MONOISOTOPIC_UNIT_MASS; 
+                firstCorrection = neuCodeLight.weighted_monoisotopic_mass + diff_integer * Lollipop.MONOISOTOPIC_UNIT_MASS; 
```

Comment:
Note to self -- let's put a comment here why we changed this: when we have some corrected components and some that aren't we get some mismatches. Using the WMM makes sure they are all corrected the same.